commands-yml/commands/web/window/set-window.yml

```diff
@@ -69,7 +69,7 @@ endpoint:
     - name: session_id
       description: ID of the session to route the command to
   json_parameters:
-    - name: name
```

Comment:
Thank you!

How about modifying like below?
Mjsonwp can work with changing test environment 

```
- name: name
   type: string
   description: (For MJSONWP) The window to change focus to
- name: handle
   type: string
   description: (For W3C) The window to change focus to
```

Reply:
Updated!